ruuvi-collector.properties.example

```diff
@@ -48,13 +48,15 @@
 # dummy           = Logs the measurements to the log rather than sending them anywhere, feasible for testing and development
 #storage.method=influxdb
 
-# Values to store, the collector is capable of calculating additional values from the values received from a tag, for example absolute humdity and dew point
-# Valid values "raw" and "extended"
-# raw      = Save only the raw values received from the tag
-# extended = In addition to the above, calculate additional values such as absolute humidity, dew point, total acceleration and air density
-#            NOTE: the influxdb_legacy storage method does not support extended values
+# Values to store, the collector is capable of calculating additional values from the values received from a tag, for example absolute humidity and dew point
+# Valid values "raw", "extended", "whitelist", "blacklist":
+# raw       = Save only the raw values received from the tag. Ignores the storage.values.list property.
+# extended  = In addition to the above, calculate additional values such as absolute humidity, dew point, total acceleration and air density.
+#             Ignores the storage.values.list property. NOTE: the influxdb_legacy storage method does not support extended values
+# whitelist = Only the values given in the storage.values.list property are stored. (influxdb_legacy storage method not supported.)
+# blacklist = Only the values not given in the storage.values.list property are stored. (influxdb_legacy storage method not supported.)
 #storage.values=extended
-
+#storage.values.list = accelerationX,accelerationY,accelerationZ,accelerationTotal,accelerationAngleFromX,accelerationAngleFromY,accelerationAngleFromZ

```

Comment:
The commented out properties in the example file should contain the defaults, and if I'm reading the code correctly, the actual default for this is empty. Either the actual default or this should be changed for consistency.

Also, perhaps a new column "Field name" could be added to the table in MEASUREMENTS.md and a comment left here referring to that so the user don't need to read the code or guess the names of the fields?

Reply:
Thanks for your comments! Adding a new column to MEASUREMENTS.md is a good idea, I wanted to add that list but couldn't think of a good place. Done!

However, I disagree with the default setting thing you mentioned. Using the whitelist or blacklist with "default settings" makes no sense whatsoever. It's completely up to the user and their use case on how they might want to use the feature. A whitelist with an empty `storage.values.list` is essentially blocking everything, while a blacklist with an empty list is the same as extended values, i.e. everything is allowed. In my case I just wanted to disable the acceleration readings from some stationary tags, but why have "raw", "extended" and "stationary" when you can have a nice generic solution like this? But I did add some explanatory texts both on MEASUREMENTS.md and ruuvi-collector.properties.example, let me know what you think. :)

Reply:
You should still add a reference to the MEASUREMENTS.md here ("see MEASUREMENTS.md for field names" or something), as not everyone reads it and some may be wondering where to find the field name list.

You are right in that an empty list makes no sense, but the commented lines in this file should reflect the defaults (by common conventions, and the mention on the first line of this file). Therefore a sensible default (that's consistent with the example properties file) would probably make the most sense, perhaps one that saves only temperature, humidity and pressure, which are probably the most commonly used values (and the only values that are available on all data formats), or the acceleration related values like this file has now? ie. have

    #storage.values.list = temperature,humidity,pressure

in this file and initialize the storage values Set (if I recall correctly) with these values if this property is not defined, rather than an empty set like it is now by default. 

The way it is now could be equally misleading as now if someone enables the whitelist but forgets to uncomment & configure the list, they will get no measurements at all and be confused whether the collector is working at all, rather than realizing that it's working but not saving the values they want because they forgot to configure the values list

Reply:
OK, just this one more commit before I'll give up and set some default values. How about I'll just validate that configuration and prevent the application from launching when there's an empty whitelist? Comments have also been added.

Reply:
This is good. Do you have any more changes to add or shall I merge?